ui/tests/acceptance/pageobjects/helloPage.js

```diff
@@ -0,0 +1,21 @@
+const {join} = require(`path`)
+
+module.exports = {
+	url: function() {
+		return this.api.launchUrl + '/#/hello'
+	},
+
+	commands: {
+		navigateAndWaitTillLoaded: async function() {
+			const url = this.url()
+			return this.navigate(url).waitForElementVisible("@accountInput")
+		}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2 tabs.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)